articles/governance/service-groups/overview.md

```diff
@@ -36,18 +36,20 @@ The same resources can be connected to many different service groups allowing di
+
+Within the hierarchy of resources, there's a limitation of one parent resource container to many children. For example, a resource can only be a member of one resource group, or a resource group can only be a member of one subscription. Service Groups introduce a new model that allows resources or resource containers to have memberships with multiple different Service Groups. A member is any resource, resource group, or subscription that is connected to a Service Group through a new resource called "MemberOf" Relationship. The Service Group allows new scenarios where the same resources can be connected to many Service Group Trees, enabling new ways to view your data.  
+
 
 #### Example Scenarios 
 * Aggregating Health Metrics
    * Organizations with multiple applications and environments can use Service Groups to aggregate health metrics across different environments. Member resources or resource containers could be from various environments within different management groups or subscriptions, can be linked to a single Service Group providing a unified view of health metrics.
-* Creating Inventory of a specific resource type
+* Creating an Inventory of a specific resource type
     * Customers can connect all Virtual Machines or CosmosDBs to the same Service Groups to get a consolidated view of all the resources of that type in the entire environment. This capability allows a customer like a Virtual Machine Administrator to view aggregated data on all their resources no matter what subscription they live in.
 
 :::image type="content" source="./media/side-by-side.png" alt-text="Diagram showing the Management Group and Service Group Hierarchies within the Microsoft Entra Tenant" Lightbox="./media/side-by-side.png":::
 
 * Aggregating monitoring metrics 
-   * Since Service Groups don't inherit permissions to the members, customers can apply least privileges to assign permissions on the Service Groups that allow viewing of metrics. This capability provides a solution where two users can be assigned access to the same Service Group, but only one is allowed to see certain resources. 
+   * Since Service Groups don't inherit permissions from the members, customers can apply least privileges to assign permissions on the Service Groups that allow viewing of metrics. This capability provides a solution where two users can be assigned access to the same Service Group, but only one is allowed to see certain resources. 
```

Comment:
'To' is the correct direction of permissions. For example, if the user gets access at an RG, they also get access to the resources within an RG.. SG does not have that downstream permission inheritance.